client/components/chip/test/index.tsx

```diff
@@ -34,11 +47,6 @@ describe( 'Chip', () => {
 		expect( chip ).toMatchSnapshot();
 	} );
 
-	test( 'renders a primary chip by default', () => {
-		const { container: chip } = renderChip( undefined, 'Message' );
-		expect( chip ).toMatchSnapshot();
-	} );
-
```

Comment:
Removing this test felt a little weird because there's still potential for JS to not include the `type` parameter, but I checked the codebase and there's no place currently using the `Chip` component without providing `type` (or providing the wrong value for `type`), so I think this is fine.